arch/arm/configs/mako_defconfig

```diff
@@ -392,7 +400,9 @@ CONFIG_FB_MSM_WRITEBACK_MSM_PANEL=y
 CONFIG_FB_MSM_MIPI_LGIT_VIDEO_WXGA_PT_PANEL=y
 CONFIG_FB_MSM_HDMI_MSM_PANEL=y
 CONFIG_BACKLIGHT_LCD_SUPPORT=y
-# CONFIG_LCD_CLASS_DEVICE is not set
+CONFIG_LCD_CLASS_DEVICE=y
+CONFIG_BACKLIGHT_CLASS_DEVICE=y
+CONFIG_BACKLIGHT_GENERIC=y
```

Comment:
It seems to be enabling various flags below and above, could you explain that better in the commit's " [wlan] Additional configs to not have modules as other things." message please?